services/app-web/serverless.yml

```diff
@@ -74,7 +74,7 @@ custom:
         export REACT_APP_S3_DOCUMENTS_BUCKET=${self:custom.s3_documents_bucket_name}
         export REACT_APP_S3_QA_BUCKET=${self:custom.s3_qa_bucket_name}
         export REACT_APP_STAGE_NAME=${sls:stage}
-        export REACT_APP_OTEL_COLLECTOR_URL=${self:custom.application_endpoint_url}/otel
+        export REACT_APP_OTEL_COLLECTOR_URL=${self:custom.api_url}/otel
```

Comment:
This change confuses me. Can you explain this a bit like aren't these totally different urls

Reply:
Ah yeah, good catch. You're right, they're totally different in review apps, but in dev/val/prod they're the same. We weren't getting frontend logs in review apps, but I needed to turn them on for testing this. For example, in review apps it was evaluating to the cloudfront distro with the `application_endpoint_url` var, but in dev/val/prod it evaluates to the set cloudfront domain (e.g `mc-review-val.onemac.cms.gov`) which is the same as the `api_url`. 

I think it's worth keeping it on for times people want to reference frontend logs in review apps in New Relic.

Also of note, the real place we have cared about `REACT_APP_OTEL_COLLECTOR_URL` is in `app-api` where we use it to set the value to the collector of the lambda layer.